test/testDeployCli.js

```diff
@@ -68,42 +69,65 @@ describe('hlx deploy', () => {
     new CLI()
       .withCommandExecutor('deploy', mockDeploy)
       .onFail((err) => {
-        assert.equal(err, 'Missing required arguments: namespace, auth\n'
-          + 'Authentication is required. You can pass the key via the HLX_AUTH environment variable, too');
+        assert.equal(err, 'Missing required arguments: wsk-namespace, wsk-auth\n'
+          + 'Authentication is required. You can pass the key via the HLX_WSK_AUTH environment variable, too');
         done();
       })
-      .run(['deploy', '--auth secret-key']);
+      .run(['deploy', '--wsk-auth secret-key']);
 
     assert.fail('deploy w/o arguments should fail.');
   });
+
   it('hlx deploy works with minimal arguments', () => {
     new CLI()
       .withCommandExecutor('deploy', mockDeploy)
       .run(['deploy',
-        '--auth', 'secret-key',
-        '--namespace', 'hlx',
+        '--wsk-auth', 'secret-key',
+        '--wsk-namespace', 'hlx',
+      ]);
+
+    sinon.assert.calledWith(mockDeploy.withEnableAuto, true);
+    sinon.assert.calledWith(mockDeploy.withEnableDirty, false);
+    sinon.assert.calledWith(mockDeploy.withWskHost, 'runtime.adobe.io');
+    sinon.assert.calledWith(mockDeploy.withWskAuth, 'secret-key');
+    sinon.assert.calledWith(mockDeploy.withWskNamespace, 'hlx');
+    sinon.assert.calledWith(mockDeploy.withLogglyHost, 'trieloff.loggly.com'); // TODO !!
+    sinon.assert.calledWith(mockDeploy.withLogglyAuth, '');
+    sinon.assert.calledWith(mockDeploy.withTarget, '.hlx/build');
+    sinon.assert.calledWith(mockDeploy.withDocker, 'trieloff/custom-ow-nodejs8:latest');
+    sinon.assert.calledWith(mockDeploy.withPrefix, 'git-github-com-example-project-helix--master--');
+    sinon.assert.calledWith(mockDeploy.withDefault, undefined);
+    sinon.assert.calledOnce(mockDeploy.run);
+  });
+
+  it('hlx deploy works with arguments provided in environment', () => {
+    process.env.HLX_WSK_AUTH = 'sekret-key';
```

Comment:
test all other possible arguments als well

Reply:
I think that would be overkill, as the `env()` method of `yargs` is well-tested and we won't get any more functional coverage: https://github.com/yargs/yargs/blob/6ee2c82df515cb1b062e4135a5dd9c386fed2b21/test/yargs.js#L756